examples/collect/host/run-and-save-output.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: troubleshoot.sh/v1beta2
+kind: HostCollector
+metadata:
+  name: run-host-cmd-and-save-output
+spec:
+  collectors:
+    - run:
+        collectorName: "my-custom-run"
+        command: "cat $TS_INPUT_DIR/dummy.yaml"
+        args: [ "--timeout", "10m" ]
```

Comment:
This is unnecessary. `cat` will error

Reply:
yes, gonna change it to somelike bash

Reply:
This is more accurate. `$TS_INPUT_DIR` will be evaluated by the shell, not `cat`

```yaml
command: "sh"
args: [ "-c", "cat $TS_INPUT_DIR/dummy.yaml" ]
```

Reply:
done